app/javascript/hooks/queries/users/useUsers.jsx

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import axios from 'axios';
+
+export default function useUsers() {
+  return useQuery('getUsers', async () => axios.get('/api/v1/users.json', {
```

Comment:
What's this query used for?

Reply:
It's in relation with the User#index that returns all Users.
https://github.com/bigbluebutton/greenlight/pull/3446#discussion_r873982483

Reply:
See my comment there